lib/matplotlib/tests/test_legend.py

```diff
@@ -517,3 +517,14 @@ def test_shadow_framealpha():
     ax.plot(range(100), label="test")
     leg = ax.legend(shadow=True, facecolor='w')
     assert leg.get_frame().get_alpha() == 1
+
+
+def test_legend_title_empty():
+    # test that if we don't set the legend title, that
+    # it comes back as an empty string, and that it is not
+    # visible:
+    fig, ax = plt.subplots()
+    ax.plot(range(10))
+    leg = ax.legend()
+    assert leg.get_title().get_text() == ""
```

Comment:
`assert not leg.get_title().get_text()`

Reply:
I don't think we should rely on boolean-ness here.

Reply:
This is [PEP-8](https://www.python.org/dev/peps/pep-0008/#programming-recommendations) convention

> For sequences, (strings, lists, tuples), use the fact that empty sequences are false.

But I'm ok, if you want to have an explicit comparison.